temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -305,4 +305,50 @@ service CloudService {
             get: "/cloud/usage",
         };
     }
+
+   // Create an export sink
+    rpc CreateExportSink(CreateExportSinkRequest) returns (CreateExportSinkResponse) {
+        option (google.api.http) = {
+            post: "/cloud/namespaces/{namespace}/sinks",
+            body: "*"
+        };
+    }
+
+    // Get an export sink 
+    rpc GetExportSink(GetExportSinkRequest) returns (GetExportSinkResponse) {
+    option (google.api.http) = {
+            get: "/cloud/namespaces/{namespace}/sinks/{sink_name}"
+        };
+    }
+
+    // Get export sinks
+    rpc GetExportSinks(GetExportSinksRequest) returns (GetExportSinksResponse) {
+        option (google.api.http) = {
+            get: "/cloud/namespaces/{namespace}/sinks"
+        };
+    }
+
+    // Update an export sink
+    rpc UpdateExportSink(UpdateExportSinkRequest) returns (UpdateExportSinkResponse) {
+    option (google.api.http) = {
+            post: "/cloud/namespaces/{namespace}/sinks/{sink_name}",
```

Comment:
```suggestion
            post: "/cloud/namespaces/{namespace}/sinks/{name}",
```

Reply:
Curious about the choice of` /sinks/ `. Do we expect more than 1 sink? I assumed we could only have 1.

Reply:
Agree, while reading this PR I was wondering if there was a scope change that multiple sinks would be supported. 

Unless there's already an existing product requirement to support multiple sinks, I'd prefer to have a simpler API versus one that is future-proofing for a growth area that may never come. If a customer needs to sink multiple places, I think it's a reasonable expectation that they handle the fan-out themselves from a single source.

Reply:
So user will be allowed to set up 2 sinks for multi region namespace, still one sink per region, but 2 sinks in total.
 
It's under a feature flag now. Not open to customer yet but backend support has completed. Would like to hear more feedback on this.

Reply:
Ah that's a good point! In that case, `sinks` makes sense. Can we document in the API somewhere that by default export has only 1 sink? To minimize confusion